src/get-todo-config.ts

```diff
@@ -43,7 +43,7 @@ export function getTodoConfig(
     mergedConfig.warn >= mergedConfig.error
   ) {
     throw new Error(
-      'The `lintTodo` configuration in the package.json contains invalid values. The `warn` value must be less than the `error` value.'
+      `The provided TODO configuration contains invalid values. The \`warn\` value (${mergedConfig.warn}) must be less than the \`error\` value (${mergedConfig.error}).`
```

Comment:
We should (separately in a new issue) decide how we want phrasing/capitalization to work for these todo files when we refer to them. Should it be `TODOS` or maybe `todos` or `.lint-todo files` something else?

Reply:
I've been using TODO or TODOs, generally. I'm open though.